Gruntfile.js

```diff
@@ -65,4 +73,7 @@ module.exports = function (grunt) {
   // load task definitions
   grunt.task.loadTasks('tasks');
   grunt.task.loadTasks('tasks/build');
+
+ // load NPM tasks
+  grunt.loadNpmTasks('grunt-angular-translate');
```

Comment:
No modifications to this Gruntfile are necessary. Just create a file called `i18nextract.js` under `/tasks/config` with the following contents:

```
export default function (grunt) {
  return {
    default_options: {
      src: ['src/**/*.js', 'src/**/*.html'],
      lang: ['en'],
      dest: 'build/tmp'
    }
  };
};
```

The task and this config will be automatically loaded.